gaps/__main__.py

```diff
@@ -0,0 +1,4 @@
+from .cli import *
```

Comment:
Can we just import `cli`

```python
from .cli import cli
```

Reply:
If we just use cli, it will raise error: 
```TEXT
ModuleNotFoundError: No module named 'cli'
```